packages/server/src/index.ts

```diff
@@ -17,6 +17,7 @@ const launch = async () => {
   await setupEnv()
 
   const router = express()
+  router.disable('x-powered-by')
```

Comment:
Why this change? Is this related to security?

Reply:
Yes and no. It only removes the `x-powered-by: 'Express'` headers so that most people wouldn't necessarily know what framework powers our API. I know that we are an open-source project but again, it's only because it's a good practice thing. In our case it doesn't really have any negative nor positive effects.